source/getting-started/configuration.rst

```diff
@@ -1,9 +1,119 @@
+Unbound assumes that a user named "unbound" exists. You can add this user with your favourite account management tool (:command:`useradd(8)`), or disable the feature with ``username: ""`` in the config. If it is enabled, after the setup, any other user privileges are dropped and the configured username is assumed.
+
+Unbound comes with the :command:`unbound-checkconf` tool. This tool allows you to check a config for errors before loading. This tool is very convenient because if any errors are found it tells you where they are and it allows you to check this before loading Unbound to a network interface on your machine.
+
+
+Set up Remote Control
+=====================
+
+A useful functionality to enable is the use of the :command:`unbound-control` command. Enabling this allows Unbound to be controlled by using the :command:`unbound-control` command, which makes starting, stopping, and reloading Unbound easier. To enable this functionality we need to add :option:`remote-control` to the config and enable it.
+
+.. code:: bash
+
+    remote-control:
+        # enable remote-control
+        control-enable: yes
+
+        # location of the files created by unbound-control-setup
+        #server-key-file: "/usr/local/etc/unbound/unbound_server.key"
+        #server-cert-file: "/usr/local/etc/unbound/unbound_server.pem"
+        #control-key-file: "/usr/local/etc/unbound/unbound_control.key"
+        #control-cert-file: "/usr/local/etc/unbound/unbound_control.pem"
+
+To set up for using the :command:`unbound-control` command, we need to invoke the :command:`unbound-control-setup` command. This creates a number of files in the default install director directory. The default install directory is ``/usr/local/etc/unbound/unbound.conf`` on most systems, but some distributions may put it in ``/etc/unbound/unbound.conf`` or ``/etc/unbound.conf``.
+
+Apart from an extensive config file, with just about all the possible configuration options, :command:`unbound-control-setup` creates the cryptographic keys necessary for the control option. 
+
+.. code:: bash
+
+    unbound-control-setup
+
+If you use a username like ``unbound`` in the config to run the daemon, you can use sudo to create the file in that user's name, so that the server is allowed to read the keys. This also works for other users if the ``/usr/local/etc/unbound/`` directory is write-protected.
+
+.. code:: bash
+
+	``sudo -u unbound unbound-control-setup``
+
+When these steps succeed, you can now control Unbound using the :command:`unbound-control` command. Note that if you're not using the name ``unbound.conf`` in the default directory, the name (and possibly path) need to be provided when using the command using the :option:`-c` flag.
+
+Set up trust anchor
+===================
+
+To enable `DNSSEC <https://en.wikipedia.org/wiki/Domain_Name_System_Security_Extensions>`_, which we strongly recommend, we need to create a trust anchor as it ensures the integrity of the responses to your queries.
```

Comment:
It's been a while since I looked at DNSSEC but isn't the trust anchor used to *verify* the integrity of the received response rather than to ensure it? Also, to which queries are you referring? Queries made to Unbound, or queries made by Unbound?